File: urionlinejudge/1006_Average2.cpp
// Created by islam elhady on 20/01/2018.

#include <iostream>
 
using namespace std;
 
int main() {
 
     double A, B, C, total;

      scanf("%lf %lf %lf", &A, &B, &C);

      total = (A * 2.0) + (B * 3.0) + (C * 5.0);
      total /= 10;

       printf("%s = %.1lf\n", "MEDIA", total);
 
    return 0;
}

File: urionlinejudge/1006_Average2.java
// Created by islam elhady on 22/01/2018.

import java.io.IOException;
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args) throws IOException {
 
       
       float A,B,C,MED;
       Scanner s = new Scanner (System.in);
       A = s.nextFloat();
       B = s.nextFloat();
       C = s.nextFloat();
       MED = (A/10 * 2) + (B/10 * 3) + (C/10 * 5);
       System.out.printf("MEDIA = %.1f\n" , MED);
 
    }
 
}